### ISSUE TITLE
Stub of getters & setters not working with sandbox

### ISSUE THREAD

**Issue by vitalets** [sinonjs/sinon#781]
When I stub getter using `sinon.stub()` everything works fine.
But when I use `sandbox.stub()` it does not work. 
Example:

``` js
var o = {
    get foo() { return 'foo' },
    get bar() { return 'bar' }
};

// via sinon
sinon.stub(o, 'foo', { get: function () { return 'foo stubbed' }});
console.log(o.foo); // foo stubbed

// via sandbox
var sandbox = sinon.sandbox.create();
sandbox.stub(o, 'bar', { get: function () { return 'bar stubbed' } });
console.log(o.bar); // Uncaught TypeError: Cannot set property bar of #<Object> which has only a getter
```

jsfiddle: http://jsfiddle.net/exzc5m3e/

Relates to: https://github.com/cjohansen/Sinon.JS/pull/692

Is it possible to support getters & setters in sandboxes?

Thanks!

**Comment by mroderick** [comment]
Thank you for reporting this.

> Is it possible to support getters & setters in sandboxes?

That's a good question. We'll need more investigation to determine if that's possible.

**Comment by mikedaly** [comment]
I recently added sinon to a project, to augment Jasmine, specifically because it supported mocking properties. I would use this feature as soon as it was available.

**Comment by nunofgs** [comment]
I just hit this issue as well.

**Comment by kuraga** [comment]
The same issue

**Comment by amrish7** [comment]
Just ran into this issue

**Comment by fatso83** [comment]
As the label says: help is much appreciated. If someone deeply craves this feature do know that we are open for pull requests ;-)

**Comment by mmoss** [comment]
@fatso83 I'll give adding some units tests and fixing this a go... fingers crossed I can get it working :smile:

**Comment by mantoni** [comment]
@mmoss Make sure to pull again. [The unit tests have been converted to Mocha](https://github.com/sinonjs/sinon/pull/1004).

**Comment by mroderick** [comment]
This is implemented with #1416
